install/etcd/etcd.yaml

```diff
@@ -0,0 +1,36 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: master-etcd
+  namespace: kube-system
+  labels:
+    openshift.io/control-plane: "true"
+    openshift.io/component: etcd
+spec:
+  restartPolicy: Always
+  hostNetwork: true
+  containers:
+  - name: etcd
+    image: openshift/origin:latest
+    workingDir: /var/lib/etcd
+    command: ["/bin/sh", "-c"]
+    args:
+    - |
+      #!/bin/sh
+      set -o allexport
+      exec openshift start etcd --config=/etc/origin/master/master-config.yaml
```

Comment:
this will be hypershift right?

Reply:
> this will be hypershift right?

once it is ready, yes.  For now, this is what we have to start.